### ISSUE TITLE
`Download the latest installer` button downloads arm64 package on Intel Macs

### ISSUE THREAD

**Issue by somelinguist** [jgm/pandoc-website#64]
I just got around to updating to the lastest version and discovered that the `Download the latest installer for macOS` button downloads the latest `*-arm64-macOS.pkg` on my Intel Mac. I didn't notice until I ran the installer and it failed saying it couldn't be installed on my Mac without further explanation. Then I noticed the file name and figured out what happened. I've since downloaded and installed the latest `*-x86_64-macOS.pkg` from the Github releases page. 

However, I wonder if the `Installing Pandoc` page on the website can be updated to detect the correct Mac architecture  automatically?

\*Thanks for the Apple Silicon builds, by the way. One of these days I'll get a new Mac and greatly appreciate it. :)

**Comment by jgm** [comment]
@mb21 you contributed downloadInstallerBtn.js -- any idea how to properly detect the architecture?
(Also needed for linux, since we offer two archictectures there too.)

**Comment by mb21** [comment]
Yeah, I remember I looked into that for PanWriter.com as well, and there was no reliable way to detect ARM vs Intel macs on all browsers at that time, so I just added two buttons: https://github.com/mb21/panwriter-website/commit/04760ddfdfa88d3bdc14a61c32760bd62d9246d0

Seems this is still the case: https://stackoverflow.com/questions/65146751/detecting-apple-silicon-mac-in-javascript
